docs/api/internal.json

```diff
@@ -0,0 +1,108 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "WP Job Manager APIs (internal)",
+    "version": "1.0",
+    "description": "Internal WP Job Manager endpoints. These endpoints are not subject to any backwards compatibility guarantees. They may change at any time without warning since are considered as internal APIs.",
+    "contact": {
+      "name": "WPJobManager",
+      "url": "https://wpjobmanager.com/"
+    }
+  },
+  "servers": [
+    {
+      "url": "http://wpjm.local"
+    }
+  ],
+  "tags": [
+    {
+      "name": "promoted-jobs"
+    }
+  ],
+  "paths": {
+    "/wp-json/wpjm-internal/v1/promoted-jobs": {
+      "get": {
+        "summary": "Promoted Jobs - Get list of promoted jobs",
+        "description": "Get promoted jobs list",
+        "operationId": "get-promotedjobs-list",
+        "tags": [
+          "promoted-jobs"
+        ],
+        "parameters": [
+          {
+            "name": "Accept",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "default": "application/json"
+            },
+            "description": "application/json"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "jobs": {
+                      "type": "array",
+                      "items": {
+                        "type": "object",
+                        "properties": {
```

Comment:
Maybe we could already add the salary field here? Just in case we decide to iterate and use it in the future.

Reply:
Added 808c8896